addons/mail/static/src/components/file_uploader/file_uploader.js

```diff
@@ -81,12 +75,14 @@ export class FileUploader extends Component {
     /**
      * @private
      * @param {Object} param0
-     * @param {Composer} param0.composer
      * @param {FileList|Array} param0.files
-     * @param {Thread} param0.thread
      * @returns {Promise}
      */
-    async _performUpload({ composer, files, thread }) {
+    async _performUpload({ files }) {
+        const composer = this.fileUploaderView.composerView && this.fileUploaderView.composerView.composer; // save before async
+        const thread = this.fileUploaderView.thread; // save before async
+        const chatter = this.fileUploaderView.attachmentBoxView && this.fileUploaderView.attachmentBoxView.chatter; // save before async
+        const activity = this.fileUploaderView.activityView && this.fileUploaderView.activityView.activity; // save before async
```

Comment:
This is not pretty, but I guess it's better that for now, until further refactor soon.

Reply:
Indeed, I preferred minimal diff to get ready for OWL 2, but this component should be completely removed if possible to have the upload input in a fragment.